radiomics/shape.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, inputImage, inputMask, **kwargs):
       self.SurfaceArea = self._calculateSurfaceArea()
 
   def _calculateSurfaceArea(self):
+    self.maskArray = self.maskArray.astype('int')  # needed for the interpolate function to work correctly
```

Comment:
@jcfr, Why did you push this commit? If you look at L42, you can see that the datatype is already forced to `int`...